src/components/slide-toggle/slide-toggle.html

```diff
@@ -0,0 +1,9 @@
+<div class="md-container">
+  <div class="md-bar"></div>
+  <div class="md-thumb-container">
+    <div class="md-thumb"></div>

```

Comment:
For the slide-toggle, we want to drive the component with an underlying native checkbox, similar to what @robertmesserle is doing in #415 (since the slide-toggle is pretty much just a checkbox with a different appearance). 

This gives you the accessibility and keyboard interaction for free.